commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/address.jsp

```diff
@@ -123,38 +123,24 @@ long commerceRegionId = BeanParamUtil.getLong(currentCommerceAddress, request, "
 	Liferay.provide(
 		window,
 		'<portlet:namespace />addStreetAddress',
-		function() {
-			var A = AUI();
+		function <portlet:namespace />addStreetAddress() {
+			const A = AUI();
+			const addStreetFields = A.one('.add-street-fields');
+			const addStreetLink = A.one('.add-street-link');
 
-			var addStreetFields = A.one('.add-street-fields');
-
-			if (addStreetFields) {
-				addStreetFields.show();
-			}
-
-			var addStreetLink = A.one('.add-street-link');
-
-			if (addStreetLink) {
-				addStreetLink.hide();
-			}
+			if (addStreetFields) addStreetFields.show();
+			if (addStreetLink) addStreetLink.hide();
```

Comment:
I don't think `if` statements like this are allowed :/. Pretty sure we require the brackets and newlines.

Reply:
https://in.liferay.com/web/global.engineering/wiki/-/wiki/Core+Development+Main/Peer+Reviews+Checklist#section-Peer+Reviews+Checklist-Formatting+for+conditions,+loops,+functions+(or+anything+that+has+a+pair+of+curly+braces)